src/test/java/org/testcontainers/containers/TarantoolCartridgeBootstrapFromLuaWithFixedPortsIT.java

```diff
@@ -23,7 +23,7 @@
  * @author Alexey Kuzin
  */
 @Testcontainers
-public class TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest {
+public class TarantoolCartridgeBootstrapFromLuaWithFixedPortsIT {
```

Comment:
I don't see changes we have discussed

You're right that we can use IT without Test and that's ok for idea regex.
But I'd like to have unification about this question. To have the same things in our every product